### ISSUE TITLE
OpenVPN error with OpenVPN on iOS

### ISSUE THREAD

**Issue by MagicFab** [angristan/openvpn-install#127]
I am testing OpenVPN profiles created with this script on an iPad 4 mini, using the OpenVPN app 1.1.1 build 212 (iOS 64 bit).

The .ovpn file imports fine but when attempting to connect I get this error:

`OpenVPN error:  PolarSSL: error parsing config private key : PKCS5 - Requested encryption or digest alg not available`

I've modified the original script to require a password for the client, in line 113:

`./easyrsa build-client-full $CLIENT #nopass`

Other than that the same profile works on a PC (Windows or Debian).

**Comment by angristan** [comment]
Does it work without your modification?

Try to remove tls-cipher or cipher from the client config file, IIRC the iOS app just sucks and isn't up-to-date.

**Comment by MagicFab** [comment]
It does work if **nopass** is left as an option at line 113:

`./easyrsa build-client-full $CLIENT nopass`

When removing nopass so there is a PK password asked in the iOS OpenVPN app, I get the above error with and without the tls-cipher line. I get the same error when removing the cipher line too.

**Comment by MagicFab** [comment]
I found a hint [here](https://github.com/OpenVPN/easy-rsa/issues/147#issuecomment-326798429):

> [the] OpenVPN [iOS App] Connect uses a different library, PolarSSL, now known as mBed TLS. OpenSSL suports PKCS#5, PolarSSL doesn't.

**Comment by MagicFab** [comment]
I found [this post](https://www.reddit.com/r/raspberry_pi/comments/6w0dxs/openvpnpivpn_on_raspbian_stretch_error_parsing/dm93nb3/) which details steps to convert the encrypted private key part of the .ovpn file to RSA (as I understand it), it worked for me.

The steps indicate how to re-encrypt the private key, it involves modifying the ovpn file to include the generated key. 

I tested this on a PC using Debian 9 and it also worked.
